ci/download.workflow

```diff
@@ -0,0 +1,13 @@
+workflow "Download" {
+  resolves = "download"
+}
+action "download" {
+  uses = "./download"
+  secrets = ["ZENODO_API_TOKEN"]
```

Comment:
do we need the token to download files? If not, then we can remove it from the `download` action

Reply:
When no `ZENODO_FILES` value is provided, in order to retrieve the list of files via the API I need the access token. [This](https://github.com/popperized/zenodo/blob/1de606ea43184dd216ea649d857f91e22a7ddf01/download/entrypoint.sh#L9) thing will not work without an access token.